docs/proposals/2017-04-20 - Statistic Series.md

```diff
@@ -0,0 +1,161 @@
+# Problem
+
+Ellipsis needs to render statistics in a series with supplementary text. (This is the replacement for the timeline and rail templates in Methode.)
+
+# Proposal
+
+A content element dedicated to displaying a single set of related statistics in a series.  Each statistic can be labelled and described, and laid out in a horizontal or vertical manner.
+
+# Example
+
+
+https://www.washingtonpost.com/sports/nationals/ryan-zimmerman-returns-as-nationals-top-marlins-for-90th-win-3-2/2014/09/20/ff3a05b8-40ff-11e4-9587-5dafd96295f0_story.html?utm_term=.bc80951132c7
+
+```
+{
+  "_id": "AAAAAAAAAAABCDEFGH",
+  "type": "statistic_series",
+
+  "direction": "vertical",
+
+  "citation": {
+    "type": "text",
+    "content": "Source: The Washington Post"
+  },
+
+  "items":
+  [
+    {
+      "label": {
```

Comment:
I think the key thing here is that this is *not* a table.  It's a data series, which may be rendered in a table, but other than the direction, there is no implicit formatting.